ingest-app/docs/api.md

```diff
@@ -1169,6 +1169,10 @@ For NRT Notification subscriptions to be used there are three new fields that ar
         </ul>
 </ul>
 
+**IMPORTANT: There is limit to creating and updating subscriptions related to the specific scenario highlighted below:
+Currently, if User1 creates a subscription to Coll1 and then User2 creates a different subscription to the same Coll1 for the same queue, it will overwrite the first subscription for that specific queue.
```

Comment:
```suggestion
Currently, if <User One> creates a subscription to <Collection 1> and then <User 2> creates a different subscription to the same <Collection 1> for the same queue, it will overwrite the first subscription for that specific queue.
```